ajax/libs/tsparticles/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "tsparticles",
+  "version": "1.6.0",
```

Comment:
```suggestion
```

Please do not include a version attr by default, the bot will populate this automatically once it starts pulling in versioned files for the library.